lib/info.js

```diff
@@ -53,6 +53,14 @@ module.exports = function getInfo(link, options, callback) {
 
   // Try getting config from the video page first.
   var url = VIDEO_URL + id;
+  
+  // Add Language Support
+  if (options.lang) {
+    url = url + "&hl=" + options.lang;
+  } else {
+    url = url + "&hl=en";
+  }
+  
```

Comment:
What I meant was, to append above on line 55, it's small enough to fit on one line anyway.